angular-hub/src/content/events/2023-12-07-angular-melbourne.md

```diff
@@ -4,4 +4,8 @@ location: Melbourne, Australia
 date: 2023-12-07
 url: https://www.meetup.com/angular-melbourne/events/297244000
 logo: assets/logos/official-logo.webp
+tags:
+  - Free
+  - Online
```

Comment:
As far as i can understand, it's not an online event.

Reply:
yeah, this paragraph tricked me :D
```
----- Connect with us after the event -----
📹 [Youtube](https://www.youtube.com/c/AngularAustralia)
```